### PR TITLE
Modified census bug.

### PR DESCRIPTION
**Pull request by jamesmcclain**
This fixes a bug in the computation of modified censuses.  A new test has also been added to exercise this fix.

Connects WikiWatershed/model-my-watershed#701

The bug in https://github.com/WikiWatershed/model-my-watershed/pull/713 that I thought was in the MMW backend python code that I had written turned out to be in the TR-55 implementation.

**To test**
   * The symptom of the bug appears in MMW when you:
       * Draw a large area of forest over a paved area (like in the middle of Philadelphia)
       * The runoff will go way down, as it should
       * Draw a bmp that cuts the forest into two pieces
       * The runoff will spring back up to its original value (because the modified census was not being correctly created)
   * You see examples of the bug by running the census given in the new test `test_create_modified_census_4` through the code before this change.  You will see that the modified census is incorrect.  Doing so after applying the change gives a correct modified census.

### CODE REVIEW

File: tr55/model.py [hunk 1]
```diff
@@ -241,15 +241,16 @@ def create_modified_census(census):
...
+                    parent['distribution'] = {cell: {'cell_count': n}}
+
+                parent['distribution'][cell]['cell_count'] -= m
+                parent['distribution'][changed_cell] = {'cell_count': m}
 
     return mod
 
 
-def simulate_water_quality(tree, cell_res, fn, current_cell=None, precolumbian=False):
+def simulate_water_quality(tree, cell_res, fn, current_cell=None, precolumbian=False):  # noqa
```
Comment (mmcfarland) [inline tr55/model.py]: If these `noqa` comments are due to line length complaints, you can always break the arguments to another line.  I'd also be ok with increasing the line length lint settings to > 80.

**Comment by mmcfarland** [comment]
+1 Tested.

File: tr55/model.py (hunk 1 above)
Comment (kdeloach) [inline tr55/model.py]: +1 For wrapping lines. I often have 2 editors side-by-side and 80 lines is the sweet spot. PEP 8 has some great guidelines around line wrapping.

Comment (jamesmcclain) [inline tr55/model.py]: Okay, I will break the lines.

**Pull request merged**